common-adapter-moshi/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
There is no need for this file nor the `consumer-rules.pro` above☝️ . Proguard rules should probably be defined in the app consuming these libraries.

Reply:
Fixed in cb30a2d8773d8303c1fb68fcd7dc1bd13b0571ce